### ISSUE TITLE
Integer overflow when determining large time to ground

### ISSUE THREAD

**Issue by zacharyburnett** [UMDBPP/PacketRaven#27]
```
2020-11-05 21:10:36,884 | INFO     | received 1 new packet(s) from https://api.aprs.fi/api/get: [APRSPacket(from_callsign='KC3HHV-7', to_callsign='APDR16', time=datetime.datetime(2020, 11, 5, 21, 10, 29), x=-76.851, y=39.16533, z=64.9224, crs=CRS.from_epsg(4326), path=['TCPIP*', 'qAC', 'FIFTH'], timestamp='1604628629', symbol='Z', symbol_table='/', comment='UMDBPP Recovery')]
2020-11-05 21:10:36,970 | INFO     | sending 1 packet(s) to postgres://128.8.202.13:5432/nearspace/testing: [APRSPacket(from_callsign='KC3HHV-7', to_callsign='APDR16', time=datetime.datetime(2020, 11, 5, 21, 10, 29), x=-76.851, y=39.16533, z=64.9224, crs=CRS.from_epsg(4326), path=['TCPIP*', 'qAC', 'FIFTH'], timestamp='1604628629', symbol='Z', symbol_table='/', comment='UMDBPP Recovery')]
Exception in Tkinter callback
Traceback (most recent call last):
  File "c:\program files\python38\lib\tkinter\__init__.py", line 1883, in __call__
    return self.func(*args)
  File "c:\program files\python38\lib\tkinter\__init__.py", line 804, in callit
    func(*args)
  File "c:\program files\python38\lib\site-packages\client\gui.py", line 500, in retrieve_packets
    parsed_packets = retrieve_packets(self.__connections, self.__packet_tracks, self.database, self.output_filename, self.start_date,
  File "c:\program files\python38\lib\site-packages\client\retrieve.py", line 82, in retrieve_packets
    if packet_track.time_to_ground >= timedelta(seconds=0):
  File "c:\program files\python38\lib\site-packages\packetraven\tracks.py", line 68, in time_to_ground
    return timedelta(seconds=self.packets[-1].coordinates[2] / current_ascent_rate)
OverflowError: Python int too large to convert to C int
```

**Comment by zacharyburnett** [comment]
https://github.com/UMDBPP/PacketRaven/blob/27f090a4201b9cbe8560616ce9cfcaf5c969d47b/packetraven/tracks.py#L94-L107

**Comment by zacharyburnett** [comment]
https://github.com/pyproj4/pyproj/issues/616

**Comment by zacharyburnett** [comment]
fixed with new descent model in 7b586d9
